GTHC/web
# Hour Dashboard Component

### ### Hour Widget 

**Purpose**
The hour widget will track the hours and nights of each member in the group. The widget allows a member to toggle between the hour and total view to see the number of hours spent and nights slept in Kville for that week or in total for tenting. There also is a dropdown that allows a user to sort the members by Hours or Night within a particular view.
 
Currently, the widget displays mock data and does not feed actual hours and nights of the tent members into the display. This PR is only front-end focused, and the backend will be done later on.

**Testing**
1. Sign in with an account and navigate to the dashboard
2.  Change View from Week to Total to see the scripts of each card change and order to the rank already selected
3. Toggle between different Ranks in order to sort the member cards in a particular order 

**Screenshots (If Needed)**
<img width="1259" alt="screen shot 2018-12-25 at 12 09 06 pm" src="https://user-images.githubusercontent.com/44485851/50427791-0e87a300-0865-11e9-8381-2b1b9e96d892.png">
